.github/workflows/klayout.yml

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Please, add the full license header to the new files, and add yourself as an author to those and the others you modified (if you want to). The authorship and code ownership will be preserved in the git history, so adding the name is cosmetic/optional.

Reply:
For my repos I limit all possible redundant info, so I only include only SPDX license indentifier. Will add the license header here.
I am fine with authorship only tracked by git.